src/display.jl

```diff
@@ -396,7 +396,7 @@ end
 
 
 # This has to be overloaded by the backend for its screen type.
-function colorbuffer(x)
+function colorbuffer(x; kwargs...)
     error("colorbuffer not implemented for screen $(typeof(x))")
```

Comment:
can we make the supported kw args explicit with docs? :)

Reply:
It's listed just below on 411. But I'll add it explicitly everywhere 👍🏻